hoomd/md/ForceComposite.cc

```diff
@@ -1048,6 +1054,45 @@ void ForceComposite::updateCompositeParticles(uint64_t timestep)
         }
     }
 
+void ForceComposite::findRigidCenters()
+    {
+    ArrayHandle<unsigned int> h_tag(m_pdata->getTags(), access_location::host, access_mode::read);
+    ArrayHandle<unsigned int> h_rtag(m_pdata->getRTags(), access_location::host, access_mode::read);
+    ArrayHandle<unsigned int> h_body(m_pdata->getBodies(),
+                                     access_location::host,
+                                     access_mode::read);
+
+    m_rigid_center.resize(m_pdata->getN() + m_pdata->getNGhosts());
+    m_lookup_center.resize(m_pdata->getN() + m_pdata->getNGhosts());
+    ArrayHandle<unsigned int> h_rigid_center(m_rigid_center,
+                                             access_location::host,
+                                             access_mode::overwrite);
+    ArrayHandle<unsigned int> h_lookup_center(m_lookup_center,
+                                              access_location::host,
+                                              access_mode::overwrite);
+    unsigned int num_centers = 0;
+    unsigned int n_particles_local = m_pdata->getN() + m_pdata->getNGhosts();
+    for (unsigned int idx = 0; idx < n_particles_local; idx++)
```

Comment:
Conventionally, we use prefix increment in for loops. There is a historic reason for this that isn't relevant anymore, but postfix increment may look like an anti-pattern to some people.

```suggestion
    for (unsigned int idx = 0; idx < n_particles_local; ++idx)
```

Reply:
There are other instances of postfix increments in `ForceComposite.cc` in code that we haven't changed. Should those be changed as well?

Reply:
No, only in the code that you have added / modified. We try not to have unrelated changes in a PR.